dev-aws/kafka-shared-msk/energy-platform/__backend.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   backend "s3" {
     bucket  = "uw-dev-pubsub-tf-applier-state"
-    key     = "dev-aws/kafka-shared-msk-energy-budget-plan"
+    key     = "dev-aws/kafka-shared-msk-energy-platform"
```

Comment:
FYI: you'll need to add a new TF module for this to apply: https://github.com/utilitywarehouse/kafka-cluster-config/tree/main/prod-aws/kafka-shared-msk#set-up-the-terraform-applier-for-this-terraform-team-module